### DISCUSSION TITLE
Best Fitbit device for showing BG on wrist? Will Charge 5 work?

### DISCUSSION THREAD

**Discussion by munster-shug** [NightscoutFoundation/xDrip#2718] [Q&A]
Hi folks,
I would like to show my BG on my wrist.
I like my Fitbit Charge 4 and I run Xdrip+ on my android phone.
(I haven;t figured out the easiest way to replace Nightscout yet)

I have a gift voucher for Fitbit....

Which fitbit device do people recommend for reading BG on wrist?  Would the Charge 5 work?
Any suggestions appreciated.!
Alan

**Comment by Navid200** [comment]
My hope has been that these forums will become busy with lots of participation.  
There is a shortage of participation with watches unfortunately.
Do you have a facebook account?  If yes, give this group a try:  https://www.facebook.com/groups/xDripG5

**Comment by XARA11** [comment]
https://www.amazon.com/Fitbit-Exercise-Equipment-Membership-Porcelain/dp/B0CC63GZ3R?th=1&linkCode=ll2&tag=smart230b8-20&linkId=4643aef8ba6f9f64a860a988a32a6c85&language=en_US&ref_=as_li_ss_tl[](url)
